donate/routes.py

```diff
@@ -143,8 +143,10 @@ def donation():
             params['email'])
         app.logger.debug("Charge created: {}".format(charge_data))
     except StripeError as error:
-        app.logger.error("StripeError: ")
-        flash(error)
+        err = error.json_body.get('error', {})
+        msg = err.get('message', {})
```

Comment:
I would replace `err.get('message', {})` with `err.get('message', "")` since it looks to me like we want a string, not a dictionary to be used with `flash`.